src/core/public/chrome/README.md

```diff
@@ -0,0 +1,127 @@
+
+##    Chrome Service
+
+- [About!](#about-)
+- [Nav Controls Service](#navcontrolsservice-)
+- [Nav Links Service](#navlinksservice-)
+- [Recently Accessed Service](#recentlyaccessedservice-)
+- [Doc Title Service](#doctitleservice-)
+- [UI](#ui-)
+
+## About :
+- **Signature** - `export interface ChromeStart`
+
+The chrome service is a high level UI service that is part of CoreStart (Core services exposed to the Plugin start lifecycle) and offers other plugins a way to add navigation controls to the UI, edit the document title, manipuate navlinks on global header as well as edit the Recent accessed tab. it consists of these sub-services/components.
+- NavControlsService : for registering new controls to be displayed in the navigation bar.
+- NavLinksService : for manipulating nav links.
+- RecentlyAccessedService : for recently accessed history.
+- DocTitleService: for accessing and updating the document title
+- UI : All the UI components,icons for e.g  header, loaders.
+
+
+- How to access ? add in interface `chrome: ChromeStart && chrome.servicesName => e.g chrome.docTitle.method`
+- Where it is getting Registered/Executed [Staring Point](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.1/src/core/public/rendering/rendering_service.tsx)
```

Comment:
Can we make the format more consistent?
add question mark after each question (L23 and L24), just like what you did for the first one `How to access?`?